code/pizzabot.js

```diff
@@ -1,7 +1,138 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+// Declaring variables for the pizzas on the menu & the price
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
+const kebab = "Kebab Pizza";
+
+let pizzaPrice;
+
+// Welcome message - Pizza-chat box 2
+document.getElementById("welcome").innerHTML = `On our menu we have ${vegetarian}, ${hawaiian}, ${pepperoni} and ${kebab}.`;
+
+// Declaring variables
+let orderName; 
+
+// Makes Pizza-chat box 3 + User-chat box 2 visible. Function is called in next function.
+let displayOrderQuantityBox = () => {
+    const orderQuantityBox = document.getElementById("orderQuantityBox");
+    const selectQuantityBox = document.getElementById("selectQuantityBox");
+
+    orderQuantityBox.classList.remove("invisible");
+    orderQuantityBox.classList.add("visible");
+    selectQuantityBox.classList.remove("invisible");
+    selectQuantityBox.classList.add("visible");
+    }
+
+// Main order-function to select pizza sort + invokes function above
+let orderPizza = (name) => {
+    orderName = name;
+    document.getElementById("howMany").innerHTML = `OK, how many ${orderName}(s) do you want?`;
+    
+    // Show progress-bar when clicking on a pizza-sort
+    const progressBar = document.getElementById("progressBar");
+    progressBar.classList.remove("invisible");
+    progressBar.classList.add("visible");
+
+    // Push header-section down when progress-bar is shown
+    const headerBox = document.querySelector("header");
+    headerBox.classList.add("marginTop");
+    
+    // Add pizza-name + first progress-box to progress-bar
+    document.getElementById("topBarPizza").innerHTML = `${orderName}`;
+    document.getElementById("progressOne").style.background = "#01DC3E";
+
+    displayOrderQuantityBox()
+}
+
+// Counting-function to select amount of pizzas
+let quantityFunction = (event) => {
+    event.preventDefault();
+
+    // Get quantity from user input-counter 
+    let orderQuantity = document.getElementById("quantityUser").value;
```

Comment:
I don't see any place where you would overwrite this value as well, `let` -> `const`